app/kubemci/cmd/getstatus.go

```diff
@@ -66,7 +68,7 @@ func NewCmdGetStatus(out, err io.Writer) *cobra.Command {
 
 func addGetStatusFlags(cmd *cobra.Command, options *GetStatusOptions) error {
 	// TODO(nikhiljindal): Add a short flag "-p" if it seems useful.
-	cmd.Flags().StringVarP(&options.GCPProject, "gcp-project", "", options.GCPProject, "[required] name of the gcp project")
+	cmd.Flags().StringVarP(&options.GCPProject, "gcp-project", "", options.GCPProject, "[optional] name of the gcp project")
```

Comment:
Description as copied from that other PR:
```name of the gcp project. Is fetched using gcloud config get-value project if unset here```
Use the same here?

Reply:
Done. (seems like it'd read a little better as "It's fetched using...", but might as well keep it consistent.)